waypoints:
  rack_a:  [1.50, 0.45, 0.0]   # x,y,yaw
  rack_b:  [1.50, 1.00, 0.0]
  rack_c:  [1.50, 1.55, 0.0]
  rack_d:  [1.50, 2.10, 0.0]
  dest:    [0.30, 1.00, 0.0]   # 卸货区